processing_r/processing/algorithm.py

```diff
@@ -291,6 +294,9 @@ def process_parameter_line(self, line):
                 self.error = self.tr('This script has a syntax error.\n'
                                      'Problem with line: {0}').format(line)
 
+        if self.save_output_values:
+            self.output_values_filename = QgsProcessingUtils.generateTempFilename('processing_values.txt')
```

Comment:
Could you defer the filename generation to execute_r_algorithm ? We don't need to generate this name until we actually run the algorithm

Reply:
I will do necessary.